src/scripts/components/Tooltip.jsx

```diff
@@ -195,7 +171,19 @@ class Tooltip extends React.Component {
    * @return {void}
    */
   openTooltip(tooltip) {
-    tooltip.timeline.play();
+    const { delay } = this.props;
+
+    // Convert delay to a number
+    // Takes care of multiple PropTypes
+    const delayAmount = Number.isInteger(delay) ? Number(delay) : 1000;
```

Comment:
If `delay` is an integer, then you shouldn't need to convert it to a Number, right?

Reply:
So delay can either be a `bool` (which will default to that `1000`) or an integer, to override that default amount.